### PR TITLE
[Docs](Job)Add job Docs

### PR DESCRIPTION
**Pull request by CalvinKirs**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/en/docs/sql-manual/sql-functions/table-functions/job-task.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+| parameter  | description | type   | required |
+|:-----|:------------|:-------|:---------|
+| type | job type    | string | yes      |
+
+***type only support mv or insert***
+
+##### Insert tasks
+The table schema of `tasks("type"="insert");` tvf：
+```
+mysql> desc  function tasks("type"="insert");
```
Comment (morningman) [inline docs/en/docs/sql-manual/sql-functions/table-functions/job-task.md:57]:
```suggestion
mysql> desc function tasks("type"="insert");
```

File: docs/en/docs/sql-manual/sql-functions/table-functions/job-task.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+#### syntax
+
+`tasks("type"="insert");`
+**parameter description**
+
+| parameter  | description | type   | required |
+|:-----|:------------|:-------|:---------|
+| type | job type    | string | yes      |
+
+***type only support mv or insert***
```
Comment (morningman) [inline docs/en/docs/sql-manual/sql-functions/table-functions/job-task.md:52]: type 支持的类型：

- insert：insert into 类型的任务。
- mv：异步物化视图产生的任务。

File: docs/zh-CN/docs/sql-manual/sql-functions/table-functions/job-task.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+#### syntax
+
+`tasks("type"="insert");`
+**参数说明**
+
+| 参数名  | 说明     | 类型     | 是否必填 |
+|:-----|:-------|:-------|:-----|
+| type | 作业类型   | string | 是    |
+
+***type 当前仅支持 insert 或者 mv***
```
Comment (morningman) [inline docs/zh-CN/docs/sql-manual/sql-functions/table-functions/job-task.md:52]: 段落和段落之间，标题和标题之间增加空行

File: docs/zh-CN/docs/sql-manual/sql-functions/table-functions/job-task.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+##### Insert tasks
+tasks("type"="insert");表结构：
```
Comment (morningman) [inline docs/zh-CN/docs/sql-manual/sql-functions/table-functions/job-task.md:54]:
```suggestion
`tasks("type"="insert")` 的表结构：
```

**Review by LemonLiTree — APPROVED** [review]
LGTM

**Review by JNSimba — APPROVED** [review]

**Comment by CalvinKirs** [comment]
run buildall

**Comment by CalvinKirs** [comment]
run feut

**Comment by CalvinKirs** [comment]
run buildall

**Review by JNSimba — DISMISSED** [review]

**Comment by CalvinKirs** [comment]
run buildall

**Review by JNSimba — APPROVED** [review]

**Pull request merged**